### DISCUSSION TITLE
Reactive LogoutHandler

### DISCUSSION THREAD

**Discussion by danpoland** [micronaut-projects/micronaut-security#1868] [Q&A]
Is there anyway to make a [LogoutHandler](https://micronaut-projects.github.io/micronaut-security/latest/api/io/micronaut/security/handlers/LogoutHandler.html) reactive?

I am using Hibernate Reactive in my project and I need to make some database calls before returning a response from my LogoutHandler.

**Comment by sdelamo** [comment]
we will need to add a new api. I created an issue https://github.com/micronaut-projects/micronaut-security/issues/1879
